### ISSUE TITLE
In-home streaming L2L with Rocket League starts with wrong window size

### ISSUE THREAD

**Issue by mooninite** [ValveSoftware/steam-for-linux#4732]
#### Your system information

* Steam client version: Oct 13 2016, at 00:48:25
* Distribution (e.g. Ubuntu): Fedora 25 x86_64
* Opted into Steam client beta?: [Yes/No] No
* Have you checked for system updates?: [Yes/No] Yes

#### Please describe your issue in as much detail as possible:
When starting Rocket League via in-house streaming the game has a loading splash screen about 500x500 pixels large. After the loading screen the game goes full screen. When the game goes full screen the Steam streaming client goes full screen, too, but at a 500x500 size bottom left corner of the game, which hides 75% of the game screen. If I open the steam overlay the streaming client corrects itself and displays the full game screen.

I can reproduce this on NVIDIA and Intel hosts and clients. It is driver independent. The cause of the problem seems to be the loading splash that Rocket League paints before the full game window is drawn.

#### Steps for reproducing this issue:

1.  Launch rocket league via in-house streaming on a Linux host and a Linux client.
2.  See loading splash. Wait for main menu of the game. Only ~25% of the game screen is displayed.
3.  Press SHIFT+TAB to bring up the Steam overlay and the full game is displayed.

**Comment by mooninite** [comment]
I updated all my machines to the new Steam client December 12th, 2016 update. The issue still exists.

**Comment by kisak-valve** [comment]
Hello @mooninite, are you still experiencing this issue on an up to date system?

**Comment by mooninite** [comment]
Yes. I just tested it with the October 2, 2019 client on the same systems. The behavior is slightly different, but the end result is the same.

Behavior now: Launching the game displays the entire hosts desktop. The Rocket League splash is displayed. Once the game loads the full screen window I can see the entire window contents briefly, but then the Steam remote desktop client resizes the output back to the 500x500 pixel size as it was before. I have to use the SHIFT+TAB trick on the host to get the remote streaming session to resize back to the full window size.

**Comment by gcaliber** [comment]
This issue exists for me for XCOM: Chimera Squad. Much of the screen in the actual game is cut off when trying to use remote play. This is from a linux host running the game with Proton to a windows client.
